src/main/scala/uk/gov/nationalarchives/db/users/Lambda.scala

```diff
@@ -18,9 +18,19 @@ class Lambda {
     databaseName match {
       case "consignmentapi" => createConsignmentApiUsers
       case "keycloak" => createKeycloakUser
+      case "bastion" => createBastionUser
     }
   }
 
+  def createBastionUser: Boolean = {
+    val user = createConsignmentApiUser(lambdaConfig.bastionUser)
+    //Grant access to tables created before we started using the migrations user
+    sql"GRANT SELECT ON ALL TABLES IN SCHEMA public TO $user;".execute().apply()
+    sql"SET ROLE migrations_user;".execute().apply()
+    //Grant access to tables created by the migrations user.
```

Comment:
Comment is also about migrations user.

Reply:
Yep, the migrations_user created the tables because flyway uses that user.